### ISSUE TITLE
HdfFile is not API

### ISSUE THREAD

**Issue by Mailaender** [jamesmudd/jhdf#365]
**Describe the bug**

I get 

> Access restriction: The type 'HdfFile' is not API (restriction on required library '~/.m2/repository/io/jhdf/jhdf/0.6.5/jhdf-0.6.5.jar')

**To Reproduce**

The first line of the README in fact:

```java
try (HdfFile hdfFile = new HdfFile(file)) {
```

I used @eclipse-m2e to add it to the `.target` platform

```xml
	<location includeDependencyDepth="none" includeSource="true" missingManifest="generate" type="Maven">
		<dependencies>
			<dependency>
				<groupId>io.jhdf</groupId>
				<artifactId>jhdf</artifactId>
				<version>0.6.5</version>
				<type>jar</type>
			</dependency>
		</dependencies>
	</location>
```

**Expected behavior**

No access rule error.

- @eclipse 2022-03
- `jhdf` 0.6.5
- Temurin 11
- @openSUSE Linux

**Comment by jamesmudd** [comment]
Thanks for opening this issue. I'm not exactly clear what the issue is. I tried creating a project in eclipse 2022-3 with Java 11 and imported the jhdf lib ok and it worked correctly. So i'm assuming I haven't reproduced your setup. Would you be able to zip up and attach your eclipse workspace? Or give a bit more detail on how you are running the app. Are you running in OSGi environment?

**Comment by Mailaender** [comment]
Yes, I tried to add it to https://github.com/OpenChrom/openchrom/blob/develop/openchrom/releng/net.openchrom.targetplatform/net.openchrom.targetplatform.target

**Comment by jamesmudd** [comment]
Think I have reproduced the issue and the cause is a mistake in the `Export-Package` manifest header. There is a PR #367 which  I think fixes the issue. Here is a test jar with the headers changed [jhdf-0.6.6-osgi-beta.zip](https://github.com/jamesmudd/jhdf/files8413162/jhdf-0.6.6-osgi-beta.zip) (rename `.zip` to `.jar`) Would be great if you could test it and let me know if this fixes this issue.

**Comment by Mailaender** [comment]
I get
> Not Found

on the link.

**Comment by jamesmudd** [comment]
Sorry try this one (not sure what happened there)
[jhdf-0.6.6-osgi-beta.zip](https://github.com/jamesmudd/jhdf/files/8416566/jhdf-0.6.6-osgi-beta.zip)

**Comment by Mailaender** [comment]
I can confirm that fixes it.

**Comment by jamesmudd** [comment]
Released v0.6.6 including this fix
